src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/PublicationViewModel.cs

```diff
@@ -16,6 +16,8 @@ public record PublicationViewModel
 
     public bool IsSuperseded { get; init; }
 
+    public PublicationSupersededByViewModel SupersededBy { get; init; } = new();
```

Comment:
This should be optional since it's set from the `PublicationCacheViewModel` where it's optional.